host.json

```diff
@@ -2,14 +2,25 @@
   "version": "2.0",
   "logging": {
     "logLevel": {
-      "default": "Information"
+      "default": "Information",
+      "DurableTask.AzureStorage": "Warning",
+      "DurableTask.Core": "Warning",
+      "Function.GetAllBonusActivations": "Warning",
+      "Function.GetBonusActivation": "Warning"
```

Comment:
`"Function.GetBonusActivation": "Warning"`
not sure that this function is a good idea change log level to `warning`

Reply:
We're not getting activations anymore, though

Reply:
The most important to disable is `GetAllBonusActivations`
For me we can remove `GetBonusActivation` from host.json

Reply:
removed not used function